deploy/nuget/prepare_dal_nuget.sh

```diff
@@ -101,6 +110,9 @@ create_package() {
         if [ "${platform}" = "linux-x64" ]; then
             dl_postfix=.so.${major_binary_version}.${minor_binary_version}
             sl_postfix=.a
+        elif [ "${platform}" = "linux-aarch64" ]; then
```

Comment:
Do we need this? or we should generalize to just linux? as per my understanding this would be arch agnostic postfixes?